### PR TITLE
Hoist signin and signup; Spash pages for world and space; Onboarding; Account details

### PR DESCRIPTION
**Pull request by mike-lvov**

### CODE REVIEW

File: src/hooks/worlds/useOnboardedWorlds.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+          COLLECTION_USERS,
+          userId,
+          COLLECTION_ONBOARDED_WORLDS
+        ),
+        where(FIELD_IS_ONBOARDED, "==", true)
+      ).withConverter(withIdConverter<UserOnboardedWorld, WorldId>()),
+    [userId]
+  );
+
+  console.log({ memoizedQuery });
```
Comment (sunny-viktoryia) [inline src/hooks/worlds/useOnboardedWorlds.ts]:
```suggestion
```

File: src/components/attendee/SplashGated/SplashSpace.hoc.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
```
Comment (sunny-viktoryia) [inline src/components/attendee/SplashGated/SplashSpace.hoc.tsx]: there's an identical file: src/components/attendee/SplashSpace/SplashSpace.hoc.tsx
do we need both?

File: src/components/attendee/SplashGated/SplashSpace.tsx [hunk 3]
```diff
@@ -0,0 +1,99 @@
+import React, { useEffect } from "react";
```
Comment (sunny-viktoryia) [inline src/components/attendee/SplashGated/SplashSpace.tsx]: there's a quite similar file `src/components/attendee/SplashSpace/SplashSpace.tsx`. do we need both?

**Review by sunny-viktoryia — APPROVED** [review]
looks good to get it merged. 🙃

File: src/components/attendee/SplashGated/SplashSpace.hoc.tsx (hunk 2 above)
Comment (mike-lvov) [reply src/components/attendee/SplashGated/SplashSpace.hoc.tsx]: Nice catch! Removed in https://github.com/sparkletown/sparkle/pull/3012/commits/2a4fe1dc4ef15783ba8a9a8fe44efb9b96c19540

File: src/components/attendee/SplashGated/SplashSpace.tsx (hunk 3 above)
Comment (mike-lvov) [reply src/components/attendee/SplashGated/SplashSpace.tsx]: Deleted in https://github.com/sparkletown/sparkle/pull/3012/commits/2a4fe1dc4ef15783ba8a9a8fe44efb9b96c19540

**Pull request merged**